pages/api/events/cron/processTimelyEvents.ts

```diff
@@ -2,53 +2,57 @@ import { NextApiRequest, NextApiResponse } from 'next';
 import { sendEmailsForTriggerAndEntity } from 'lib/events/consumers/userNotifications/emails/emails';
 import { getLiquidatedLoansForTimestamp } from 'lib/events/timely/timely';
 import { captureException, withSentry } from '@sentry/nextjs';
-import { configs } from 'lib/config';
+import { Config, devConfigs, prodConfigs } from 'lib/config';
+import { onRinkeby } from 'lib/chainEnv';
```

Comment:
@adamgobes chainEnv doesn't work in a multi-chain environment

Reply:
seems like this should be coming from the env, anyway? Dev vs. prod?

Reply:
ok, i can modify, we can just run liquidations for all configs including rinkeby

Reply:
and to @wilsoncusack's point, yes. we just need to know dev vs prod as shown below. but we don't even really need this, can just run for all envs